modules/ROOT/pages/_partials/attributes.adoc

```diff
@@ -53,3 +53,14 @@
 :xamarin-sdk: AeroGear SDK for Xamarin
 
 :mobile-developer-console: Mobile Developer Console
+
+// links 
+
+:link-product-doc-home: https://docs.aerogear.org/aerogear/latest/
```

Comment:
This is just an opinion, feel free to ignore: it is easier to read the "{link-product-doc-home}/device-security" than "{link-product-doc-home}device-security" - the location of the '/'

Downstream:
:link-product-doc-home: https://{cp-server}/documentation/{language}/{DocInfoProductName}
:idm-guide-link: {link-product-doc-home}/{DocInfoProductNumber}/{paging}/using_the_identity_management_mobile_service

Upstream:
:link-product-doc-home: https://docs.aerogear.org/aerogear/latest/
:idm-guide-link: {link-product-doc-home}identity-management

Reply:
yes, that's better